### ISSUE TITLE
Log loss function value

### ISSUE THREAD

**Issue by smartinellimarco** [TorchBoard/PyTorchBoard#10]
Make the TorchBoard API able to capture and store loss values.

Acceptance criteria:
1. The API must take a value each step.
2. Use steps and values to plot how it evolves.
3. It should be able to handle multiple losses (use many keys).
